### ISSUE TITLE
Collection languages

### ISSUE THREAD

**Issue by baloodusudouest** [Radarr/Radarr#6911]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Hi,

Add Radarr to support the collection in different languages like they do it for the movie?

For exemple, the fantastics beats collection 
https://www.themoviedb.org/collection/435259-fantastic-beasts-collection?language=en
In french it's "les animaux fantastiques"
https://www.themoviedb.org/collection/435259-fantastic-beasts-collection?language=fr

### Describe the solution you'd like

We can change the language in the settings of the app, and it's work for the name of the movie, if it can be the same for the collection

### Describe alternatives you've considered

Add the hability to change the collection name automatically with the TMDB, or edit it manually

### Anything else?

no

**Comment by bakerboy448** [comment]
What's the issue / where?

You can edit the collection/list name already.

**Comment by RedWings-R** [comment]
Yes could be a great feature if the collection name in the nfo match the language selected for metadata of movie.

(Emby for example duplicates collection because no id is provide for the collection in nfo)
